apis/accounts.es6.js

```diff
@@ -1,27 +1,26 @@
-import BaseEndpoint from '../apiBase/BaseEndpoint';
-import Account from '../models/account';
+import { runQuery } from '../apiBase/APIRequestUtils';
+import Account from '../models2/account';
 
-export default class AccountsEndpoint extends BaseEndpoint {
-  move = this.notImplemented('move');
-  copy = this.notImplemented('copy');
-  put = this.notImplemented('put');
-  patch = this.notImplemented('patch');
-  post = this.notImplemented('post');
-  del = this.notImplemented('del');
 
-  path(method, query={}) {
-    if (query.user === 'me') {
-      return 'api/v1/me';
-    } else {
-      return `user/${query.user}/about.json`;
-    }
+const getPath = (query) => {
+  if (query.user === 'me') {
+    return 'api/v1/me';
```

Comment:
Do we also need to support the logged-out scenario (as with the v1 code here: https://github.com/reddit/node-api-client/blob/master/old-src/endpoints/v1.es6.js#L1039), so that we can inspect features/variants? Or should we have a separate mechanism for that in 3X?